java/gradle.java/test/unit/data/artifacts/shadowed/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
forgot to mention that I think this is fine to downgrade specific modules.

But we should not forget about this, since they could stay behind and cause problems later. Ideally we should try to keep the versions in sync or close together.

Reply:
Yes, we shall keep that in our mind.

Telling the truth, these test are on the border line of unit/integration, as they are using third-party plugins. Those plugins are not necessary support the current Gradle release.